partcad/src/partcad/project.py

```
@@ -231,69 +236,61 @@
-                    map(
-                        lambda project_name: self.name + "/" + project_name,
-                        imports,
+                dependencies = list(filtered)
+            if absolute:
+                children.extend(
+                    list(
+                        map(
+                            lambda project_name: self.name + "/" + project_name,
+                            dependencies,
+                        )
                     )
```

Ruff (0.8.2)

Unnecessary `map()` usage (rewrite using a list comprehension)

Replace `map()` with a list comprehension

(C417)